metric_providers/gpu/energy/nvidia/smi/component/provider.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, resolution, skip_check=False):
 
 
     def check_system(self):
-        super().check_system()
+        self.check_parallel_provider_running()
```

Comment:
Because I'm overwriting the check_system with a seperate check - (which nvidia-smi etc...)

As that check is a one-liner, we can rewrite the base.py system_check to take a variable, but that won't cover all of our bases anyways, so I don't see the benefit